nfcclient/app.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from time import sleep
+import requests
+
+from config import ClientConfig
+from nfc_reader import NFCReader
+
+logging.basicConfig(
+    format='%(asctime)s %(levelname)-8s %(message)s',
+    level=logging.INFO,
+    datefmt='%Y-%m-%d %H:%M:%S')
+
+try:
+    import RPi.GPIO as GPIO
+except ImportError:
+    print('No GPIO library found')
+    GPIO = None
```

Comment:
Shouldn't you exit the program with error instead of setting None?

Reply:
Did that so i would be able to run this locally.